PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/PushImageAction.java

```diff
@@ -50,22 +52,25 @@
 
     @Override
     public void onActionPerformed(AnActionEvent event) {
-        Project project = event.getProject();
-        if (project == null) {
+        Module module = DataKeys.MODULE.getData(event.getDataContext());
+        if (module == null) {
             notifyError(Constant.ERROR_NO_SELECTED_PROJECT);
             return;
         }
-        ApplicationManager.getApplication().invokeLater(() -> runConfiguration(project));
+        Project project = module.getProject();
+        ApplicationManager.getApplication().invokeLater(() -> runConfiguration(module));
     }
 
     @SuppressWarnings({"deprecation", "Duplicates"})
-    private void runConfiguration(Project project) {
+    private void runConfiguration(Module module) {
+        Project project = module.getProject();
         final RunManagerEx manager = RunManagerEx.getInstanceEx(project);
         final ConfigurationFactory factory = configType.getPushImageRunConfigurationFactory();
         RunnerAndConfigurationSettings settings = manager.findConfigurationByName(
-                String.format("%s: %s", factory.getName(), project.getName()));
+                String.format("%s: %s:%s", factory.getName(), project.getName(), module.getName()));
```

Comment:
need space before the last %s?

Reply:
I removed the whitespace intentionally, to combine projectName & moduleName as a group. Do you think we should reserve the space ?

Reply:
just make sure it is as designed. No need to reserve the space